crates/sovereign-sdk/full-node/db/sov-db/src/ledger_db/rpc.rs

```diff
@@ -59,9 +61,9 @@ impl LedgerRpcProvider for LedgerDB {
 
     fn get_soft_batch(
         &self,
-        batch_id: SoftBatchIdentifier,
+        batch_id: &SoftBatchIdentifier,
     ) -> Result<Option<SoftBatchResponse>, anyhow::Error> {
-        let batch_num = self.resolve_soft_batch_identifier(&batch_id)?;
+        let batch_num = self.resolve_soft_batch_identifier(batch_id)?;
```

Comment:
what does self.resolve_soft_batch_identifier return?

if it is something line Num/Hash

lets update this function

Reply:
returns batch_num